### PR TITLE
Fix Documentation Bugs

### LINKED ISSUE #141: [PE-D][Tester B] Wrong example provided for add command in the User Guide
**Steps to Reproduce:** 
1. Paste this whole example into the command Box

**Expected:**  Should **not** show an error as it is included in the User Guide

**Actual:** Shows an `error` 

**Also**, the examples are too lengthy. Maybe try making it shorter.

-------------
Labels: `severity.Medium` `type.DocumentationBug`
original: gauravuj/ped#3

### LINKED ISSUE #154: [PE-D][Tester A] Unrelate command exists but is not included in the documentation
-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: Alteqa/ped#9

### LINKED ISSUE #163: [PE-D][Tester E] Lack of information on usage
What will the notification look like and how do I know if it is going to work?

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: wolffe88/ped#13

### LINKED ISSUE #165: [PE-D][Tester E] Documentation should replace placeholder with actual keywords
Instead of role/ROLE, it should be role/employee, role/supplier and role/client for each of the add command to remove ambiguity.

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: wolffe88/ped#14

### LINKED ISSUE #166: [PE-D][Tester A] User guide example list has role parameter as r/ instead or role/
Adding john doe with role as r/employee instead of role/employee

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: Alteqa/ped#13

### LINKED ISSUE #174: [PE-D][Tester E] Documentation for edit does not include all the possible fields that i can edit
I am able to change the documentation field for edit but it is not reflected in the user guide

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: wolffe88/ped#16

### PR DESCRIPTION
**Pull request by nuyer**
Fix documentation bugs (User guide) except those issues that need updated UI

Fix #141 Fix wrong command example
Fix #154 Include Unrelated Command in User guide
Fix #163 Remove Birthday Feature
Fix #165 Edit examples when adding each role contact
Fix #166 Fix wrong prefix
Fix #174 add fields to edit command details

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -169,7 +174,7 @@ Format: `list`
 
 Edits an existing person in the address book.
 
-Format: `edit i/ID [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [role/ROLE] [t/TAG]…​`
+Format: `edit i/ID [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [role/ROLE] [t/TAG] [r/REMARK] [dept/DEPARTMENT] [job/JOBTITLE] [skills/SKILLS] [prod/PRODUCTS] [tos/TERMSOFSERVICE] [pref/PREFERENCES]`
```
Comment (purivirakarin) [inline docs/UserGuide.md]: I don't think we should show all these field for a person (any role)

Instead we may not include the fields specific to each role, and add three more line to include a role-specific prefixes

File: docs/UserGuide.md [hunk 2]
```diff
@@ -330,7 +339,6 @@ NetConnect data are saved automatically as a JSON file `[JAR file location]/data
 If your changes to the data file makes its format invalid, NetConnect will discard all data and start with an empty data file at the next run.  Hence, it is recommended to take a backup of the file before editing it.<br>
 Furthermore, certain edits can cause the NetConnect to behave in unexpected ways (e.g., if a value entered is outside the acceptable range). Therefore, edit the data file only if you are confident that you can update it correctly.
 
-</section>
```
Comment (purivirakarin) [inline docs/UserGuide.md]: is this belong to `line 333`?

**Review by purivirakarin — CHANGES_REQUESTED** [review]
Thanks for the changes, can see some comments I have left.

File: docs/UserGuide.md (hunk 2 above)
Comment (nuyer) [reply docs/UserGuide.md:333]: one of the issue i receive ask me to specify if I don't understand wrongly #174

File: docs/UserGuide.md (hunk 1 above)
Comment (nuyer) [reply docs/UserGuide.md]: one of the issue i receive ask me to specify if I don't understand wrongly https://github.com/AY2324S2-CS2103T-F12-1/tp/issues/174

**Review by purivirakarin — APPROVED** [review]

**Pull request merged**